### ISSUE TITLE
[FROZEN] Iteration Plan for Spring 2020

### ISSUE THREAD

**Issue by KexyBiscuit** [AOSC-Dev/aosc-os-abbs#2124]
This plan captures our work in **Spring**.

## Important dates

- May 1, 2020: Start first survey for updates. 
- June 15, 2020: Freeze Testing-Proposed.
    - Merge all changes to Testing for updates.
    - Transfer all skip-ahead changes to Experimental.
    - All non-complete entries on Testing-Proposed => Testing to be deferred to Spring 2020.
- June 15, 2020: Commence draft on Summer 2020 Iteration Plan.
- June 30, 2020: Merge all changes to Stable, cycle completion.

### Stable-Proposed => Stable

TBD.

### Testing-Proposed => Testing

TBD.

## Priority Tasks

Tasks listed in the following sections will be subject to prioritised processing. These items must be complete before the cycle completes, and is therefore recommended that packagers work on these entries before others.

### Stable-Proposed => Stable

- [x] January 2, 2020: Pillow v6.2.2 [6.2.2](https://pillow.readthedocs.io/en/stable/releasenotes/6.2.2.html)
  - [x] `pillow`
  - *Contains security updates.* (#2167) 
- [x] March 4, 2020: Django v3.0.4 [Django 3.0.4 release notes](https://docs.djangoproject.com/en/dev/releases/3.0.4/)
  - [x] `django`
  - *Contains security updates.* (#2164) 
- [x] March 5, 2020: KDE Applications v19.12.3 [Release 19.12.3](https://kde.org/announcements/releases/2020-03-apps-update/)
  - [x] `groups/kde-applications`
- [x] March 30, 2020: Apache Httpd v2.4.43
  - [x] `httpd`
  - *Contains security updates.* (#2163) 
- [x] March 31, 2020: GnuTLS v3.6.13
  - [x] `gnutls`
  - [x] `gnutls+32`
  - *Contains security updates.* (#2174) 
- [x] April, 2020: MATE Desktop v1.24.1 (some components)
  - [x] `groups/mate`
- [x] April 8, 2020: Linux Kernel v5.6.3 [ChangeLog-5.6.3](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.6.3)
  - [x] `linux-kernel`
  - [x] `linux+kernel`
  - [x] `kernel-tools`
  - *Contains security updates.* (#2143)
- [x] April 23, 2020: Ceph v13.2.10 [v13.2.10 Mimic released](https://ceph.io/releases/v13-2-10-mimic-released/)
  - [x] `ceph`
  - *Contains security updates.* (#2166)
- [x] April 24, 2020: `ydcv` v0.7
  - *Contains critical usability fix.*
- [x] April 28, 2020: OpenLDAP v2.4.50 [2.4.50 Release Changes](https://www.openldap.org/software/release/changes.html)
  - [x] `openldap`
  - [x] `openldap+32`
  - *Contains security updates.* (#2144)
- [x] May 5, 2020: Plasma Desktop v5.18.5 [Plasma 5.18.5](https://kde.org/announcements/plasma-5.18.5.php)
  - [x] `groups/plasma`
- [x] May 5, 2020: Thunderbird v68.8.0 [Thunderbird — Release Notes (68.8.0) — Thunderbird](https://www.thunderbird.net/en-US/thunderbird/68.8.0/releasenotes/)
  - [x] `thunderbird`
  - *Contains security updates.* (#2147)
- [x] May 6, 2020: Glade v3.36.0 [glade 3.36.0](https://mail.gnome.org/archives/ftp-release-list/2020-May/msg00014.html)
  - [x] `glade`
  - *Contains critical usability fix.*
- [x] May 7, 2020: Rust v1.43.1 [Announcing Rust 1.43.1](https://blog.rust-lang.org/2020/05/07/Rust.1.43.1.html)
  - [x] `rustc`
  - *Contains security updates.* (#2169) 
- [x] May 8, 2020: Firefox v76.0.1 [Firefox 76.0.1, See All New Features, Updates and Fixes](https://www.mozilla.org/en-US/firefox/76.0.1/releasenotes/)
  - [x] `firefox`
  - *Contains security updates.* (#2146)
- [x] May 8, 2020: FreeRDP v2.1.0 [2.1.0 released](http://www.freerdp.com/2020/05/08/2_1_0-released)
  - [x] `freerdp`
  - *Contains security updates.* (#2149)
- [x] May 8, 2020: Pale Moon v28.9.3 [Pale Moon 28.9.3](https://github.com/MoonchildProductions/Pale-Moon/releases/tag/28.9.3_Release)
  - [x] `palemoon`
  - *Contains security updates.* (#2175)
- [x] May 12, 2020: ClamAV v0.102.3
  - [x] `clamav`
  - *Contains security updates.* (#2159) 
- [x] May 13, 2020: Apache Ant v1.10.8 [Apache Ant 1.9.15 and 1.10.8](https://ant.apache.org/)
  - [x] `apache-ant`
  - *Contains security updates.* (#2181)
- [x] May 13, 2020: LibEXIF (revision only) [USN-4358-1](https://usn.ubuntu.com/4358-1/)
  - [x] `libexif`
  - [x] `libexif+32`
  - *Contains security updates.* (#2151)
- [x] May 13, 2020: Squid (revision only) [USN-4356-1](https://usn.ubuntu.com/4356-1/)
  - [x] `squid`
  - *Contains security updates.* (#2152) 
- [x] May 14, 2020: APT (revision only) [USN-4359-1](https://usn.ubuntu.com/4359-1/)
  - [x] `apt`
  - *Contains security updates.* (#2150)
- [x] May 14, 2020: JSON-C (revision only) [USN-4360-1](https://usn.ubuntu.com/4360-1/)
  - [x] `json-c`
  - [x] `json-c+32`
  - *Contains security updates.* (#2153) 
- [x] May 14, 2020: OpenConnect v8.10 [Changelog](https://www.infradead.org/openconnect/changelog.html)
  - [x] `openconnect`
  - *Contains security updates.* (#2162) 
- [x] May 14, 2020: PHP v7.4.6 [PHP 7.4.6 Released!](https://www.php.net/archive/2020.php#2020-05-14-3)
  - [x] `php7`
  - *Contains security updates.* (#2173) 
- [x] May 18, 2020: Dovecot v2.3.10.1 [\[Dovecot-news\] v2.3.10.1 released](https://dovecot.org/pipermail/dovecot-news/2020-May/000437.html)
  - [x] `dovecot`
  - *Contains security updates.* (#2157) 
- [x] May 19, 2020: BIND v9.16.3
  - [x] `bind`
  - *Contains security updates.* (#2156) 
- [x] May 19, 2020: Chromium, Google Chrome v83.0.4103.61 [Stable Channel Update for Desktop](https://chromereleases.googleblog.com/2020/05/stable-channel-update-for-desktop_19.html)
  - [x] `chromium`
  - [x] `google-chrome`
  - *Contains security updates.* (#2177)
- [x] May 19, 2020: Exim (revision only) [USN-4366-1](https://usn.ubuntu.com/4366-1/)
  - [x] `exim`
  - *Contains security updates.* (#2158)
- [x] May 20, 2020: FreeRDP v2.1.1 [2.1.1](https://github.com/FreeRDP/FreeRDP/releases/tag/2.1.1)
  - [x] `freerdp`
  - *Contains security updates.* (#2179)
- [x] May 21, 2020: `libvirt` (revision only) [USN-4371-1](https://usn.ubuntu.com/4371-1/)
  - *Contains security updates.* (#2160) 
- [x] May 21, 2020: Qemu (revision only) [USN-4372-1](https://usn.ubuntu.com/4372-1/)
  - [x] `qemu`
  - *Contains security updates.* (#2161)
- [x] May 25, 2020: SQLite v3.32.1 [SQLite Release 3.32.1 On 2020-05-25](https://www.sqlite.org/releaselog/3_32_1.html)
  - [x] `sqlite`
  - [x] `sqlite+32`
  - *Contains security updates.* (#2193)
- [x] May 27, 2020: Unbound (revision only) [USN-4374-1](https://usn.ubuntu.com/4374-1/)
  - [x] `unbound`
  - *Contains security updates.* (#2172)
- [x] May 29, 2020: MiniUPNPD (revision only)
  - [x] `miniupnpd`
  - *Contains security updates.* (#2178)
- [x] June 1, 2020: CA Certificates (revision only) [USN-4377-1](https://usn.ubuntu.com/4377-1/)
  - [x] `ca-certs`
  - *Contains security updates.* (#2180)
- [x] June 2, 2020: Firefox v77.0.1 [MFSA2020-20](https://www.mozilla.org/en-US/security/advisories/mfsa2020-20/)
  - [x] `firefox`
  - *Contains security updates.* (#2185)
- [x] June 2, 2020: Thunderbird v68.9.0 [MFSA2020-22](https://www.mozilla.org/en-US/security/advisories/mfsa2020-22/)
  - [x] `thunderbird`
  - *Contains security updates.* (#2186)
- [x] June 3, 2020: Chromium, Google Chrome v83.0.4103.97 [Stable Channel Update for Desktop](https://chromereleases.googleblog.com/2020/06/stable-channel-update-for-desktop.html)
  - [x] `chromium`
  - [x] `google-chrome`
  - *Contains security updates.* (#2184)
- [x] June 3, 2020: Django v3.0.7 [Django security releases issued: 3.0.7 and 2.2.13](https://www.djangoproject.com/weblog/2020/jun/03/security-releases/)
  - [x] `django`
  - *Contains security updates.* (#2183)
- [x] June 3, 2020: GnuTLS v3.6.14 [GNUTLS-SA-2020-06-03](https://gnutls.org/security-new.html#GNUTLS-SA-2020-06-03)
  - [x] `gnutls`
  - *Contains security updates.* (#2187)
- [x] June 9, 2020: Flash Player PPAPI Plugin v32.0.0.387 [APSB20-30](https://helpx.adobe.com/security/products/flash-player/apsb20-30.html)
  - [x] `flashplayer-ppapi`
  - *Contains security updates.* (#2188)
- [x] June 9, 2020: Intel Microcodes v202000609 [microcode-20200609 release](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20200609)
  - [x] `intel-ucode`
  - *Contains security updates.* (#2189)
- [x] June 9, 2020: `libjpeg-turbo` (revision only) [USN-4386-1](https://usn.ubuntu.com/4386-1/)
  - [x] `libjpeg-turbo`
  - [x] `libjpeg-turbo+32`
  - *Contains security updates.* (#2190)
- [x] June 10, 2020: Linux Kernel v5.7.2 [ChangeLog-5.7.2](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.7.2)
  - [x] `linux-kernel`
  - [x] `linux+kernel`
  - [x] `kernel-tools`
  - *Contains security updates.* (#2189)
- [x] June 10, 2020: Linux Kernel (LTS) v5.4.46 [ChangeLog-5.4.46](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.46)
  - [x] `linux-kernel-lts`
  - [x] `linux+kernel+lts`
  - *Contains security updates.* (#2189)
- [ ] June 11, 2020: .NET Core (June 2020, various versions) [Microsoft Security Advisory CVE-2020-1108 | .NET Core Denial of Service Vulnerability](https://github.com/dotnet/announcements/issues/157)
  - *Contains security updates.* (#2191)
- [ ] June 11, 2020: PowerShell v6.2.6 [Microsoft Security Advisory CVE-2020-1108 | .NET Core Denial of Service Vulnerability](https://github.com/dotnet/announcements/issues/157)
  - *Contains security updates.* (#2194)

### Testing-Proposed => Testing

- [x] December 29, 2019: OpenSC v0.20.0 [OpenSC-0.20.0](https://github.com/OpenSC/OpenSC/releases/tag/0.20.0)
  - [x] `opensc`
  - *Contains security updates.* (#2165) 
- [x] December 30, 2019: Trinity Desktop Environment v14.0.7 [Trinity Desktop Environment R14.0.7 Released!](https://www.trinitydesktop.org/)
  - [x] `groups/trinity`
  - *Contains security fixes.* (#2121)
- ~April 23, 2020: KDE Applications v20.04 [April Apps Update Featuring Releases 20.04](https://kde.org/announcements/releases/2020-04-apps-update/)~
  - ~`groups/kde-applications`~
- [ ] April 24, 2020: LxQt v0.15.0 [0.15.0](https://github.com/lxqt/lxqt/releases/tag/0.15.0)
  - [ ] `groups/lxqt`
- [x] May 9, 2020: KDE Frameworks v5.70.0 [KDE Frameworks 5.70.0 released](https://kde.org/announcements/kde-frameworks-5.70.0.php)
  - [x] `groups/kde-applications`
- [x] May 15, 2020: KDE Applications v20.04.1 [May Apps Update Featuring Releases 20.04.1](https://kde.org/announcements/releases/2020-05-apps-update/)
  - [x] `groups/kde-applications`
- [x] June 9, 2020: KDE Plasma v5.19.0 [KDE Plasma 5.19.0 Released](https://kde.org/announcements/plasma-5.19.0)
  - [x] `groups/plasma`
- [x] June 13, 2020: KDE Frameworks v5.71.0 [KDE Releases Frameworks 5.71.0](https://www.kde.org/announcements/kde-frameworks-5.71.0)
  - [x] `groups/kf5`
- [ ] 2020?: GNOME v3.36.2
  - [ ] `groups/gnome`
- [ ] X11 Libraries, Fonts, Applications components update.
  - [ ] `x11-lib+32`
  - [ ] `x11-font`

### Stable-Proposed => Stable

- [ ] August 25, 2019: Aptitude v0.8.12
  - [ ] `aptitude`
- [x] August 28, 2019: `diff-so-fancy` v1.2.7
- [ ] September 5, 2019: ZeroTier One v1.4.6 [Version 1.4.6](https://github.com/zerotier/ZeroTierOne/releases/tag/1.4.6)
  - [ ] `zerotier-one`
- [ ] September 22, 2019: `icdiff` v1.9.5
- [ ] October 21, 2019: Asunder v2.9.5 [Change log](http://littlesvr.ca/asunder/releases/ChangeLog)
  - [ ] `asunder`
- [ ] October 24, 2019: VirtualGL v2.6.3 [2.6.3](https://github.com/VirtualGL/virtualgl/releases/tag/2.6.3)
  - [ ] `virtualgl`
  - [ ] `virtualgl+32`
- [ ] November 1, 2019: Texmaker v5.0.4 [Changelog](https://www.xm1math.net/texmaker/log.html)
  - [ ] `texmaker`
- [ ] November 7, 2019: Gnumeric v1.12.46 [Gnumeric 1.12.46 aka "TBD" is now available](http://www.gnumeric.org/announcements/1.12/gnumeric-1.12.46.html)
  - [ ] `gnumeric`
- [ ] November 10, 2019: `arp-scan` v1.9.7
- [x] November 11, 2019: `telepathy-qt` v0.9.8 [The "Still moving" release](https://github.com/TelepathyIM/telepathy-qt/releases/tag/telepathy-qt-0.9.8)
- [ ] November 15, 2019: Fisher v3.2.10
  - [ ] `fisherman`
  - *Provide `fisher`, as it is now renamed.*
- [ ] November 22, 2019: `restic` v0.9.6 [restic 0.9.6](https://github.com/restic/restic/releases/tag/v0.9.6)
- [ ] November 27, 2019: Zopfli v1.0.3
  - [ ] `zopfli`
- [ ] November 28, 2019: Zypper v1.14.33
  - [ ] `zypper`
- [ ] December 4, 2019: RStudio v1.2.5033
  - [ ] `rstudio`
- [x] December 6, 2019: `waifu2x-converter-cpp` v5.3.3 [waifu2x-converter-cpp v5.3.3](https://github.com/DeadSix27/waifu2x-converter-cpp/releases/tag/v5.3.3)
  - *New upstream. Exceptional update because v5.2 (current `stable`) won't build with newer CUDA.*
- [ ] December 16, 2019: `mcfly` v0.3.6 [VI keybindings](https://github.com/cantino/mcfly/releases/tag/v0.3.6)
- ~December 17, 2019: strongSwan v5.8.3 [5.8.3](https://wiki.strongswan.org/versions/75)~
  - ~`strongswan`~
- [x] December 20, 2019: BCFtools v1.10.2 [1.10.2](https://github.com/samtools/bcftools/releases/tag/1.10.2)
  - [x] `bcftools`
- [ ] December 20, 2019: TigerVNC v1.10.1 [TigerVNC 1.10.1](https://github.com/TigerVNC/tigervnc/releases/tag/v1.10.1)
  - [ ] `tigervnc`
  - [ ] December 21, 2019: Hexchat v2.14.3 [Changelog](https://hexchat.readthedocs.io/en/latest/changelog.html)
  - [ ] `hexchat`
- [ ] December 22, 2019: Stellarium v0.19.3 [Stellarium v0.19.3 has been released!](https://stellarium.org/release/2019/12/22/stellarium-0.19.3.html)
  - [ ] `stellarium`
- [ ] December 30, 2019: `ncompress` v4.2.4.6 [Various accumulated cleanups](https://github.com/vapier/ncompress/releases/tag/v4.2.4.6)
- [ ] December 30, 2019: `pngquant` v2.12.6
- [ ] December 31, 2019: PPP v2.4.8
  - [ ] `ppp`
- [ ] December 31, 2019: `ranger` v1.9.3
- [ ] January 5, 2020: `socat` v1.7.3.4 [CHANGES](http://www.dest-unreach.org/socat/doc/CHANGES)
- [ ] January 7, 2020: ZXing (C++) v1.0.8 [v1.0.8: Merge pull request #115 from marxin/add-missing-includes](https://github.com/nu-book/zxing-cpp/releases/tag/v1.0.8)
  - [ ] `zxing-cpp`
- [ ] January 9, 2020: uGet v2.2.3
  - [ ] `uget`
- ~January 9, 2020: `xapps` v1.6.10~
- ~January 13, 2020: X.Org Server v1.20.7~
  - ~`xorg-server`~
- [ ] January 16, 2020: `opam` v2.0.6 [2.0.6](https://github.com/ocaml/opam/releases/tag/2.0.6)
- [ ] January 19, 2020: KeePassXC v2.5.3 [Release 2.5.3](https://github.com/keepassxreboot/keepassxc/releases/tag/2.5.3)
  - [ ] `keepassxc`
- [ ] January 19, 2020: Midnight Commander v4.8.24
  - [ ] `mc`
- [ ] January 24, 2020: Blueman v2.1.2 [2.1.2](https://github.com/blueman-project/blueman/releases/tag/2.1.2)
  - [ ] `blueman`
- [ ] January 24, 2020: OUnit v2.2.2 [v2.2.2](https://github.com/gildor478/ounit/releases/tag/v2.2.2)
- [ ] January 25, 2020: PCI Utilities v3.6.4
  - [ ] `pciutils`
- [ ] January 27, 2020: `dhcpcd` v8.1.6
- [ ] January 27, 2020: Maxima v5.43.2
  - [ ] `maxima`
- [ ] January 29, 2020: QuiteRSS v0.19.3 [QuiteRSS 0.19.3 (29.01.2020)](https://quiterss.org/en/article/1523)
  - [ ] `quiterss`
- [ ] January 29, 2020: Yubico Authenticator for Desktop v5.0.2 [Yubico Authenticator for Desktop 5.0.2](https://github.com/Yubico/yubioath-desktop/releases/tag/yubioath-desktop-5.0.2)
  - [ ] `yubioath-desktop`
- [ ] January 29, 2020: Yubikey Manager v3.1.1 [yubikey-manager 3.1.1](https://github.com/Yubico/yubikey-manager/releases/tag/yubikey-manager-3.1.1)
  - [ ] `yubikey-manager`
- [ ] January 31, 2020: `lighttpd` v1.4.55
- [ ] January 31, 2020: `util-linux` v2.33.2
- [ ] February 1, 2020: Cgl v0.60.3
  - [ ] `cgl`
- [ ] February 1, 2020: CoinUtils v2.11.4
  - [ ] `coinutils`
- [ ] February 1, 2020: Osi v0.108.6
  - [ ] `osi`
- [ ] February 2, 2020: `ruby-cairo` v1.16.5
- [ ] February 4, 2020: Clp v1.17.5
  - [ ] `clp`
- [ ] February 9, 2020: `par2cmdline` v0.8.1 [v0.8.1: par2cmdline 0.8.1](https://github.com/Parchive/par2cmdline/releases/tag/v0.8.1)
- [ ] February 9, 2020: `ruby-pkg-config` v1.4.1
- [ ] February 10, 2020: NCurses Disk Usage v1.14.2
  - [ ] `ncdu`
- [ ] February 11, 2020: Ruby-Locale v2.1.3
  - [ ] `ruby-locale`
- [ ] February 13, 2020: HAProxy v2.0.13 [\[ANNOUNCE\] haproxy-2.0.13](https://www.mail-archive.com/haproxy@formilux.org/msg36327.html)
  - [ ] `haproxy`
- [ ] February 15, 2020: Mutt v1.13.4 [The Mutt E-Mail Client](http://www.mutt.org/)
  - [ ] `mutt`
- [ ] February 19, 2020: ALSA v1.2.2 [Changes v1.2.1.2 v1.2.2](https://alsa-project.org/wiki/Changes_v1.2.1.2_v1.2.2)
  - [ ] `alsa-lib`
  - [ ] `alsa-plugins`
  - [ ] `alsa-utils`
  - [ ] `alsa-lib+32`
  - [ ] `alsa-plugins+32`
- [ ] February 19, 2020: CryptoMiniSat v5.6.8 [CryptoMiniSat 5.6.8](https://github.com/msoos/cryptominisat/releases/tag/5.6.8)
  - [ ] `cryptominisat2`
  - [ ] Transitionalise `cryptominisat2` and move files to `cryptominisat`.
- [ ] February 19, 2020: Siege v4.0.5 [Siege 4.0.5](https://www.joedog.org/2020/02/19/siege-4-0-5/)
  - [ ] `siege`
- [ ] February 23, 2020: BOINC v7.16.5
  - [ ] `boinc`
- [ ] February 23, 2020: Claws Mail v3.17.5 [Latest News](https://www.claws-mail.org/news.php)
- [ ] February 24, 2020: CoinMP v1.8.4
  - [ ] `coin-mp`
- [x] February 22, 2020: MATE Themes v3.22.21 [mate-themes 3.22.21 release](https://github.com/mate-desktop/mate-themes/releases/tag/v3.22.21)
  - [x] `mate-themes`
- [ ] February 27, 2020: SPICE v0.14.3 [Major Changes in 0.14.3](https://gitlab.freedesktop.org/spice/spice/-/releases)
  - [ ] `spice`
- [ ] February 27, 2020: Zotero v5.0.84
  - [ ] `zotero`
- [ ] February 29, 2020: R v3.6.3 [R 3.6.3 is released](https://stat.ethz.ch/pipermail/r-announce/2020/000650.html)
  - [ ] `r`
- [ ] March 1, 2020: OpenBLAS v0.3.9 [OpenBLAS 0.3.9 version](https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.9)
  - [ ] `openblas`
- ~March 1, 2020: Pale Moon v28.8.4 [Pale Moon 28.8.4](https://github.com/MoonchildProductions/UXP/releases/tag/PM28.8.4_Release)~
  - ~`palemoon`~
- [ ] March 2, 2020: FriBiDi v1.0.9 [FriBiDi Version 1.0.9](https://github.com/fribidi/fribidi/releases/tag/v1.0.9)
  - [ ] `fribidi`
  - [ ] `fribidi+32`
- [ ] February 4, 2020: Hardware Data v0.333
  - [ ] `hwdata`
- [ ] February 9, 2020: Gost v2.11.0 [v2.11.0](https://github.com/ginuerzh/gost/releases/tag/v2.11.0)
  - [ ] `gost`
- [ ] March 3, 2020: PARI/GP v2.11.3 [Changelog](https://pari.math.u-bordeaux.fr/pub/pari/unix/pari-2.11.3.changelog)
  - [ ] `pari`
- [ ] March 8, 2020: BorgBackup v1.1.11 [Version 1.1.11 (2020-03-08)](https://borgbackup.readthedocs.io/en/stable/changes.html#version-1-1-11-2020-03-08)
  - [ ] `borgbackup`
- [ ] March 8, 2020: `pcsc-tools` v1.5.6 [Changelog](http://ludovic.rousseau.free.fr/softwares/pcsc-tools/Changelog)
- [ ] March 8, 2020: `xfce4-statusnotifier-plugin` v0.2.2
- [ ] March 8, 2020: Xfce4 Power Manager v1.6.6
  - [ ] `xfce4-power-manager`
- [ ] March 9, 2020: Cbc v2.10.5
  - [ ] `cbc`
- [ ] February 1, 2020: KCPTUN v20200321
  - [ ] `kcptun`
- [ ] March 10, 2020: `xrdp` v0.9.13 [xrdp v0.9.13](https://github.com/neutrinolabs/xrdp/releases/tag/v0.9.13)
- ~March 11, 2020: Flash Player PPAPI Plugin v32.0.0.344~
  - ~`flashplayer-ppapi`~
- [ ] March 14, 2020: Debootstrap v1.0.123
  - [ ] `debootstrap`
- [ ] March 14, 2020: ModemManager 1.12.8 [ANN: ModemManager 1.12.8](https://lists.freedesktop.org/archives/modemmanager-devel/2020-March/007729.html)
  - [ ] `modemmanager`
- [ ] March 15, 2020: Inxi v3.0.38+1
  - [ ] `inxi`
- ~March 15, 2020: Min v1.13.2 [1.13.2: Camera and audio permissions](https://github.com/minbrowser/min/releases/tag/v1.13.2)~
  - ~`min`~
- [ ] March 15, 2020: TeXworks v0.6.4 [TeXworks 0.6.4](https://github.com/TeXworks/texworks/releases/tag/release-0.6.4)
  - [ ] `texworks`
- [ ] March 18, 2020: Metasploit v5.0.81
  - [ ] `metasploit`
- [ ] March 18, 2020: Music Player Daemon v0.21.21
  - [ ] `mpd`
- ~March 18, 2020: Wireguard DKMS Modules (Kernels ~5.5) v0.0.20200318~
  - ~`wireguard-dkms`~
- [ ] March 19, 2020: Wireguard Tools v1.0.20200319
  - [ ] `wireguard-tools`
- [x] March 20, 2020: BlueZ v5.54 [Release of BlueZ 5.54 (and 5.53)](http://www.bluez.org/release-of-bluez-5-54-and-5-53/)
  - [x] `bluez`
- [ ] March 20, 2020: DeadBeef v1.8.3
  - [ ] `deadbeef`
- ~March 20, 2020: Linux Firmware v20200320 (Git snapshot) [Merge branch 'v12573.77' of https://github.com/erinlo/linux_fw_scp](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=edf390c23a4e185ff36daded36575f669f5059f7)~
  - ~`firmware-free`~
  - ~`firmware-nonfree`~
- ~March 21, 2020: Brave Browser v1.5.115 [Release Channel v1.5.115](https://github.com/brave/brave-browser/releases/tag/v1.5.115)~
  - ~`brave-browser`~
- [ ] March 21, 2020: Greybird v3.22.12 [Greybird 3.22.12](https://github.com/shimmerproject/Greybird/releases/tag/v3.22.12)
  - [ ] `greybird`
- ~March 21, 2020: Linux Kernel v5.5.11 [ChangeLog-5.5.11](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.5.11)~
  - ~`linux-kernel`~
  - ~`linux+kernel`~
  - ~`kernel-tools`~
- ~March 21, 2020: Linux Kernel (LTS) v5.4.27 [ChangeLog-5.4.27](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.27)~
  - ~`linux-kernel-lts`~
  - ~`linux+kernel+lts`~
- [x] March 21, 2020: OpenResty v1.15.8.3 [OpenResty 1.15.8.3 Released](http://openresty.org/en/ann-1015008003.html)
  - [x] `openresty`
- [ ] March 21, 2020: RetroArch v1.8.5 [RetroArch 1.8.5 released!](https://www.libretro.com/index.php/retroarch-1-8-5-released/)
  - [ ] `retroarch`
  - *Update assets.*
- [x] March 22, 2020: ImageMagick v6.9.11+2
  - [x] `imagemagick`
- [ ] March 22, 2020: Toxcore v0.2.11 [v0.2.11](https://github.com/TokTok/c-toxcore/releases/tag/v0.2.11)
  - [ ] `toxcore`
- [ ] March 23, 2020: Rambox v0.7.4 [0.7.4](https://github.com/ramboxapp/community-edition/releases/tag/0.7.4)
  - [ ] `rambox`
- [x] March 23, 2020: `you-get` v0.4.1423
- ~March 24, 2020: DNSCrypt v2.0.41 [2.0.41](https://github.com/DNSCrypt/dnscrypt-proxy/releases/tag/2.0.41)~
  - ~`dnscrypt`~
- [ ] March 24, 2020: V2Ray v4.23.1 [V4.23.1](https://github.com/v2ray/v2ray-core/releases/tag/v4.23.1)
  - [ ] `v2ray`
- ~March 24, 2020: `youtube-dl` v2020.03.24 [youtube-dl 2020.03.24](https://github.com/ytdl-org/youtube-dl/releases/tag/2020.03.24)~
- [x] March 25, 2020: Trojan v1.15.1 [trojan 1.15.1](https://github.com/trojan-gfw/trojan/releases/tag/v1.15.1)
  - [x] `trojan`
- [x] March 26, 2020: DNSCrypt Proxy v2.0.42 [2.0.42](https://github.com/DNSCrypt/dnscrypt-proxy/releases/tag/2.0.42)
  - [x] `dnscrypt`
- [x] March 29, 2020: strongSwan v5.8.4 [ChangeLog](https://www.strongswan.org/latest-changelog/)
  - [x] `strongswan`
- [ ] March 29, 2020: X.Org Server v1.20.8 [\[ANNOUNCE\] xorg-server 1.20.8](https://lists.x.org/archives/xorg-announce/2020-March/003041.html)
  - [ ] `xorg-server`
- [x] April 13, 2020: Doxygen v1.8.18 [Release 1.8.18](http://www.doxygen.nl/manual/changelog.html#log_1_8_18)
  - [x] `doxygen`
- [x] April 17, 2020: OpenVPN v2.4.9 [OpenVPN 2.4.9 - released on 2020.04.17](https://openvpn.net/community-downloads/)
  - [x] `openvpn`
- [x] April 17, 2020: XMRig v5.11.0 [xmrig 5.11.0](https://github.com/xmrig/xmrig/releases/tag/v5.11.0)
  - [x] `xmrig`
- [x] April 19, 2020: Opera v68.0.3618.63
  - [x] `opera`
- [x] April 23, 2020: Time Zone Database v2020a [\[tz-announce\] 2020a release of tz code and data available](http://mm.icann.org/pipermail/tz-announce/2020-April/000058.html)
  - [x] `tzdata`
- [x] May 3, 2020: SeaMonkey v2.53.2 [SeaMonkey 2.53.2 released](https://www.seamonkey-project.org/releases/seamonkey2.53.2/)
  - [x] `seamonkey`
- [x] May 4, 2020: Hardware Database v0.335
  - [x] `hwdata`
- [x] May 7, 2020: Vivaldi v3.0.1874.38 [Minor update (3) for Vivaldi Desktop Browser 3.0](https://vivaldi.com/zh-hans/blog/news/minor-update-3-for-vivaldi-desktop-3-0/)
  - [x] `vivaldi`
- [x] May 11, 2020: `xapps` v1.8.4
  - *Needed for stable Cinnamon.*
- [x] May 15, 2020: Min Browser v1.14.1 [1.14.1: Bug fixes and UI improvements](https://github.com/minbrowser/min/releases/tag/v1.14.1)
  - [x] `min`
- [x] May 15, 2020: Tor v0.4.3.5
  - [x] `tor`
- [x] May 19, 2020: Linux Firmwares v20200519 (Git snapshot) [Merge tag 'iwlwifi-fw-2020-05-19' of git://git.kernel.org/pub/scm/linux/kernel/git/iwlwifi/linux-firmware](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=8ba6fa665c52093ddc0d81137fc3c82cee2c5ef8)
  - [x] `firmware-free`
  - [x] `firmware-nonfree`
- [x] May 19, 2020: NetworkManager-strongSwan v1.5.2
  - [x] `networkmanager-strongswan`
- [x] May 20, 2020: Intel Microcodes v20200520 [microcode-20200520 release](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20200520)
  - [x] `intel-ucode`
- [x] May 21, 2020: Flash Player Plugin (PepperAPI) v32.0.0.371
  - [x] `flashplayer-ppapi`
- ~May 22, 2020: Thunderbird v68.8.1 [Thunderbird Release Notes](https://www.thunderbird.net/en-US/thunderbird/68.8.1/releasenotes/)~
  - ~`thunderbird`~
- [x] May 22, 2020: Zstandard v1.4.5 [Zstandard v1.4.5](https://github.com/facebook/zstd/releases/tag/v1.4.5)
  - [x] `zstd`
- [x] May 23, 2020: Gost v2.11.1 [v2.11.1](https://github.com/ginuerzh/gost/releases/tag/v2.11.1)
  - [x] `gost`
- [x] May 24, 2020: Telegram Desktop v2.1.7 [v 2.1.7](https://github.com/telegramdesktop/tdesktop/releases/tag/v2.1.7)
  - [x] `telegram-desktop`
- [x] May 25, 2020: `i2pd` v2.32.0 [2.32.0](https://github.com/PurpleI2P/i2pd/releases/tag/2.32.0)
- [x] May 25, 2020: NetSurf v3.10 [NetSurf 3.10 released](https://www.netsurf-browser.org/downloads/)
  - [x] `netsurf`
  - [x] `libnspsl` v0.1.6
  - [x] `libnslog` v0.1.3
  - [x] `libharu` v2.3.0
  - [x] `duktape` v2.5.0
  - [x] `libnsutils` v0.1.0
  - [x] `libnsbmp` v0.1.6
  - [x] `libdom` v0.4.1
  - [x] `libcss` v0.9.1
  - [x] `nsgenbind` v0.8
  - [x] `netsurf-buildsystem` v1.9
- [x] May 27, 2020: CA Certificates v20200527 (Git snapshot) [Bug 1618404 - Set CKA_NSS_SERVER_DISTRUST_AFTER for Symantec root certs](https://github.com/nss-dev/nss/commit/9a29c1a5384ddd718f2953d3f2e71fa9043bd121)
  - [x] `ca-certs`
- [x] May 27, 2020: Linux Kernel v5.6.15 [ChangeLog-5.6.15](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.6.15)
  - [x] `linux-kernel`
  - [x] `linux+kernel`
  - [x] `kernel-tools`
- [x] May 27, 2020: Linux Kernel (LTS) v5.4.43 [ChangeLog-5.4.43](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.43)
  - [x] `linux-kernel-lts`
  - [x] `linux+kernel+lts`
- [x] May 28, 2020: Brave Browser v1.9.76 [Desktop Release Channel v1.9.76](https://github.com/brave/brave-browser/releases/tag/v1.9.76)
  - [x] `brave-browser`
- [x] May 29, 2020: `gn` v20200529 (Git snapshot) [Add Ihor Karavan to list of authors](https://gn.googlesource.com/gn/+/b175fa5d6ae1640742aad593185780a08f6ba224)
- [x] May 29, 2020: `youtube-dl` v2020.05.29 [youtube-dl 2020.05.29](https://github.com/ytdl-org/youtube-dl/releases/tag/2020.05.29)
- [x] April 30, 2020: IntelliJ IDEA 2020.1.1 [IntelliJ IDEA 2020.1.1 is Avaliable!](https://blog.jetbrains.com/idea/2020/04/intellij-idea-2020-1-1/)
  - [x] `intellij-idea`
- [x] June 6, 2020: `babl` v0.1.78
- [x] June 6, 2020: GEGL v0.4.24
  - [x] `gegl-0.4`
- [x] June 11, 2020: GIMP v2.10.20 [GIMP 2.10.20 Released](https://www.gimp.org/news/2020/06/11/gimp-2-10-20-released/)
  - [x] `gimp`

### Testing-Proposed => Testing

- [x] August 22, 2017: Ephoto v1.5 [Ephoto 1.5 Released](https://www.enlightenment.org/news/ephoto-1-5-released)
  - [x] `ephoto`
- [x] June 29, 2018: Lensfun 0.3.95
  - *This thing is actively developed but rarely tagged. Might use git snapshot instead?*
- [ ] March 4, 2019: ProxyMan v3.1.2 [Bug fixes in v3](https://github.com/himanshub16/ProxyMan/releases/tag/v3.1.2)
  - [ ] `proxyman`
- [ ] July 29, 2019: Exiv2 v0.27.2 [Changelog](https://www.exiv2.org/changelog.html)
  - [ ] `exiv2`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/exiv2)._
- [ ] August 4, 2019: digiKam v6.2.0 [digiKam 6.2.0 is released](https://www.digikam.org/news/2019-08-04-6.2.0_release_announcement/)
  - [ ] `digikam`
- [ ] August 5, 2019: Volk v2.0.0 [Release 2.0.0](https://github.com/gnuradio/volk/releases/tag/v2.0.0)
  - [ ] `volk`
- [x] August 7, 2019: MDDS v1.5.0
  - [x] `mdds`
- [x] August 8, 2019: Ixion v0.15.0
  - [x] `ixion`
- [ ] August 10, 2019: GNU Radio v3.8.0.0
  - [ ] `gnuradio`
- [ ] August 14, 2019: Boost v1.71.0 [Version 1.71.0](https://www.boost.org/users/history/version_1_71_0.html)
  - [ ] `boost`
  - [ ] _Rebuild [reverse dependencies](https://packages.aosc.io/revdev/boost)._
- [ ] August 19, 2019: ASIO v1.14.0
  - [ ] `asio`
  - _Defer if breaks other packages._
- [ ] August 20, 2019: GNU Scientific Library v2.6 [GNU Scientific Library 2.6 Released](http://lists.gnu.org/archive/html/info-gnu/2019-08/msg00004.html)
  - [ ] `gsl`
- ~August 29, 2019: `liborcus` v0.15.2~
- [ ] September 3, 2019: Ceph v14.2.3
  - [ ] `ceph`
  - *Drop?*
- [ ] September 5, 2019: `libsigc++` v3.0.0 [3.0.0](https://github.com/libsigcplusplus/libsigcplusplus/releases/tag/3.0.0)
- [ ] September 19, 2019: OpenEXR v2.4.0 [v2.4.0](https://github.com/openexr/openexr/releases/tag/v2.4.0)
  - [ ] `openexr`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/openexr)._
- [ ] September 22, 2019: Nimble v0.11.0 [v0.11.0](https://github.com/nim-lang/nimble/releases/tag/v0.11.0)
  - [ ] `nimble`
- [ ] October 3, 2019: ICU v65 [ICU 65](http://site.icu-project.org/download/65)
  - [ ] `icu`
  - [ ] _Rebuild [reverse-dependencies](https://packages.aosc.io/revdep/icu)._
- ~October 6, 2019: Python-EFL v1.23.0~
  - ~`python-efl`~
- [ ] October 7, 2019: Open MPI v4.0.2 [NEWS](https://raw.githubusercontent.com/open-mpi/ompi/v4.0.x/NEWS)
  - [ ] `openmpi`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/openmpi)._
- [x] October 9, 2019: `liborcus` v0.15.3
- [ ] October 10, 2019: GEOS v3.8.0
  - [ ] `geos`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/geos)._
- [ ] October 11, 2019: Racer v2.1.28 [racer version 2.1.28](https://github.com/racer-rust/racer/releases/tag/v2.1.28)
  - [ ] `racer`
- [ ] October 20, 2019: PostGIS v3.0.0
  - [ ] `postgis`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/postgis)._
- [ ] October 24, 2019: VirtualGL v2.6.3 [2.6.3](https://github.com/VirtualGL/virtualgl/releases/tag/2.6.3)
  - [ ] `virtualgl`
  - [ ] `virtualgl+32`
- [ ] October 27, 2019: VeraCrypt v1.24+1 (v1.24-Hotfix1)
  - [ ] `veracrypt`
- ~October 28, 2019: Rspamd v2.1~
  - ~`rspamd`~
- [ ] October 29, 2019: Protocol Buffers v3.10.1 [Protocol Buffers v3.10.1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.10.1)
  - [ ] `protobuf`
  - _Rebuild [reverse-dependencies](https://packages.aosc.io/revdep/protobuf). Defer if build errors are unresolvable._
- [ ] November 8, 2019: CGAL v5.0 [CGAL-5.0](https://github.com/CGAL/cgal/releases/tag/releases%2FCGAL-5.0)
  - [ ] `cgal`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/cgal). Defer to v4.14.1 if build errors are unresolvable._
- [ ] November 9, 2019: GRASS v7.8.1 [GRASS GIS 7.8.1](https://trac.osgeo.org/grass/wiki/Release/7.8.1-News)
  - [ ] `grass`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/grass)._
- [ ] November 12, 2019: Closure Compiler v20191111
  - [ ] `closure-compiler`
- [x] November 20, 2019: Rspamd v2.2
  - [x] `rspamd`
- [ ] November 26, 2019: Julia v1.3.0 [v1.3.0](https://github.com/JuliaLang/julia/releases/tag/v1.3.0)
  - [ ] `julia`
- [ ] November 27, 2019: Poppler v0.83.0 [Poppler](https://poppler.freedesktop.org/)
  - [ ] `poppler`
  - [ ] `poppler+32`
  - _Rebuild [reverse dependencies](https://packages.aosc.io/revdep/poppler). Defer if build errors are unresolvable._
- [ ] November 30, 2019: Coq v8.10.2 [Coq 8.10.2](https://github.com/coq/coq/releases/tag/V8.10.2)
  - [ ] `coq`
- [ ] December 6, 2019: QGIS v3.10.1
  - [ ] `qgis`
- [ ] December 10, 2019: `mediastreamer` v4.3.0
- [ ] December 10, 2019: `ortp` v4.3.0
- [x] December 12, 2019: Pandoc v2.9 [pandoc 2.9](https://github.com/jgm/pandoc/releases/tag/2.9)
  - [x] `pandoc`
- [ ] December 16, 2019: RethinkDB v2.4.0
  - [ ] `rethinkdb`
- [x] February 12, 2020: PCRE v8.44
  - [x] `pcre`
  - [x] `pcre+32`
- [x] February 14, 2020: Connman v1.38 [Release 1.38](https://git.kernel.org/pub/scm/network/connman/connman.git/commit/?id=1ee420ace2b8edb0d4025f469aaa3d00d220dc98)
  - [x] `connman`
- [x] February 16, 2020: `citra` v0+git20200215 [Update to a newer commit, multiple](https://github.com/citra-emu/citra/commit/4991c0121a1aeb0930f2318a4800325574391e8e)
- [x] February 16, 2020: `mypaint-brushes` v1.3.1 [v1.3.1 - brushes usable with libmypaint >= 1.3](https://github.com/mypaint/mypaint-brushes/releases/tag/v1.3.1)
- [x] February 16, 2020: `yuzu` v0+git20200216 [Update to a newer commit, multiple](https://github.com/yuzu-emu/yuzu/commit/26006cbd2c875b472ff1e16e7529544cd03d2915)
- [x] February 18, 2020: Avahi v0.8.0 [Avahi 0.8, the 'Dobro Jutro' release](https://github.com/lathiat/avahi/releases/tag/v0.8)
  - [x] `avahi`
- [x] Feburary 23, 2020: `libgphoto2` v2.5.24
  - *Add command line utility `gphoto` (v2.5.23, Jun 2, 2019).*
- [ ] February 26, 2020: NCBI VDB v2.10.4
  - [ ] `ncbi-vdb`
- [x] March 22, 2020: GNU Automake v1.16.2 [automake-1.16.2 released \[stable\]](http://savannah.gnu.org/forum/forum.php?forum_id=9706)
  - [x] `automake`
- [x] March 26, 2020: `yuzu` v0+git20200326 [Update to a newer commit, multiple](https://github.com/yuzu-emu/yuzu/commit/4991c0121a1aeb0930f2318a4800325574391e8e)
- [x] April 8, 2020: PyCharm v2020.1 [PyCharm 2020.1 Out Now](https://blog.jetbrains.com/pycharm/2020/04/pycharm-2020-1-out-now/)
  - [x] `pycharm`
- [x] April 17, 2020: Darktable v3.0.2
  - [x] `darktable`
- [x] April 19, 2020: Pixman v0.40.0 [\[ANNOUNCE\] pixman release 0.40.0 now available](https://lists.x.org/archives/xorg-announce/2020-April/003043.html)
  - [x] `pixman`
  - [x] `pixman+32`
- [x] April 21, 2020: Nginx v1.18.0
  - [x] `nginx`
- [x] April 21, 2020: OpenSSH-HPN v8.1p1
  - [x] `openssh-hpn`
- [x] April 29, 2020: Python EFL Bindings v1.24.0
  - [x] `python-efl`
- [x] April 29, 2020: QEMU v5.0.0 [QEMU version 5.0.0 released](https://www.qemu.org/2020/04/29/qemu-5-0-0/)
  - [x] `qemu`
- [x] April 30, 2020: Edi v0.8.0 [Edi 0.8.0 Release](https://www.enlightenment.org/news/2020-04-30-edi-0.8-release)
  - [x] `edi`
- [x] May 10, 2020: Terminology v1.7.0 [Terminology 1.7.0 Release](https://www.enlightenment.org/news/2020-05-10-terminology-1.7.0)
  - [x] `terminology`
- [x] May 11, 2020: `libmypaint` v1.6.1 [1.6.1 - libtool `-release` flag no longer used](https://github.com/mypaint/libmypaint/releases/tag/v1.6.1)
- [x] May 12, 2020: Memcached v1.6.6 [ReleaseNotes166](https://github.com/memcached/memcached/wiki/ReleaseNotes166)
  - [x] `memcached`
- [x] May 22, 2020: Ardour v6.0 [Ardour 6.0 released](https://ardour.org/whatsnew.html)
  - [x] `ardour`
- [x] May 25, 2020: EFL (Enlightenment Foundation Library) [EFL 1.24.2 release](https://www.enlightenment.org/news/efl-1.24.2)
  - [x] `efl`
- [x] May 26, 2020: FlightGear v2020.1.2 [Changelog 2020.1](http://wiki.flightgear.org/Changelog_2020.1)
  - [x] `flightgear` v2020.1.2
  - [x] `simgear` v2020.1.2
- [x] May 27, 2020: OpenSSH v8.3p1 [OpenSSH 8.3 was released on 2020-05-27](https://www.openssh.com/txt/release-8.3)
  - [x] `openssh`
- [x] May 29, 2020: MyPaint v2.0.1 [MyPaint v2.0.1 (2020-05-29)](https://github.com/mypaint/mypaint/releases/tag/v2.0.1)
  - [x] `mypaint`
- [x] May 31, 2020: Enlightenment v0.24.1 [Enlightenment DR 0.24.1 Release](https://www.enlightenment.org/news/e24.1)
  - [x] `enlightenment`
- [x] June 1, 2020: Audacious v4.0.4 [Audacious 4.0.4 released](https://audacious-media-player.org/news/49-audacious-4-0-4-released)
  - [x] `audacious`
- [x] June 1, 2020: Git v2.27.0 [Git 2.27 Release Notes](https://raw.githubusercontent.com/git/git/master/Documentation/RelNotes/2.27.0.txt)
  - [x] `git`
- [x] June 2, 2020: KDevelop v5.5.2 [KDevelop 5.5.2 released](https://www.kdevelop.org/news/kdevelop-552-released)
  - [x] `kdevelop`
- [x] June 3, 2020: APT v2.1.6
  - [x] `apt`
- [x] June 3, 2020: Qt Creator v4.12.2 [Qt Creator 4.12.2 released](https://www.qt.io/blog/qt-creator-4.12.2-released)
  - [x] `qtcreator`
- [x] June 10, 2020: Mesa v20.1.1 [Mesa 20.1.1 Release Notes / 2020-06-10](https://docs.mesa3d.org/relnotes/20.1.1.html)
  - [x] `mesa`
  - [x] `mesa+32`

### v7.0.6 => v7.0.7 (Stable-Proposed => Stable)

- [x] June 3, 2017: Make (revision, backported patch) [\[SV 51159\] Use a non-blocking read with pselect to avoid hangs.](http://git.savannah.gnu.org/cgit/make.git/commit/?id=b552b05251980f693c729e251f93f5225b400714)
  - [x] `make`
- [x] March 16, 2020: Binutils v2.33 (Git branch update, `binutils-2_33-branch`) [gas, arm: Fix bad backport](https://sourceware.org/git?p=binutils-gdb.git;a=commit;h=b9414a8064c4e10c469ef5e0aebbf46c9fbdcf80)
  - [x] `binutils`
- [x] April 24, 2020: Bash v5.0.17
  - [x] `bash`
- [x] May 6, 2020: Linux Kernel API Headers v5.4.39 [ChangeLog-5.4.39](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.39)
  - [x] `linux+api`
- [x] May 9, 2020: Glibc v2.30 (Git branch update, `release/2.30`) [NEWS: Mention fixes for BZ 25810/25896/25902/25966](https://sourceware.org/git?p=glibc.git;a=commit;h=4748829f86a458b76642f3e98b1d80f7b868e427)
  - [x] `glibc`

### v7.0.7 => v7.0.8 (Stable-Proposed => Stable)

- [x] Bash v5.0.17 (revision, `/bin => /usr/bin`)
    - [x] `bash`
- [x] GCC v9.2.1-6 (revision, `/usr/lib64 => /usr/lib`, where applicable)
    - [x] `gcc-runtime`
    - [x] `gcc`

### v7.0.x => v7.1.0 (Testing-Proposed => Testing)

- [x] January 18, 2020: GMP v6.2.0 [GMP 6.2 News](https://gmplib.org/gmp6.2.html)
  - [x] `gmp`
- [x] January 19, 2020: GNU make v4.3 [GNU make 4.3 released!](https://lists.gnu.org/archive/html/info-gnu/2020-01/msg00004.html)
  - [x] `make`
- [x] February 1, 2020: Binutils v2.34 [Changes in 2.34](https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=blob_plain;f=binutils/NEWS;hb=refs/tags/binutils-2_34)
  - [x] `binutils`
  - *Track `binutils-2_34-branch` branch.*
- [x] May 19, 2020: Glibc v2.30 (Git branch update, `release/2.30`) [nss_compat: internal_end*ent may clobber errno, hiding ERANGE \[BZ #25976\]](https://sourceware.org/git/?p=glibc.git;a=commit;h=1ba9ebfce371cea8ab1cd5f1577a18d2f1a8c863)
  - [x] `glibc`
- [x] June 10, 2020: Linux Kernel API Headers v5.4.46 [ChangeLog-5.4.46](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.46)
  - [x] `linux+api`
- [x] June 13, 2020: Ncurses v6.2+20200613
  - [x] `ncurses`
- [x] June 16, 2020: GCC v9.3.1 (Git branch update, `releases/gcc-9`) [Daily bump](https://gcc.gnu.org/git?p=gcc.git;a=commit;h=93c2834e92419abf04fd9f54ca25ed22086611b0)
  - [x] `gcc`
  - [x] `gcc-runtime`

### Deferred to Core 8 (Summer 2020?)

- February 1, 2020: Glibc v2.31 [Release/2.31](https://sourceware.org/glibc/wiki/Release/2.31)
  - `glibc`

## Packages to Add

- [x] `yubikey-manager-qt` v1.1.4
- [ ] `monero` v0.15.0.5
- [ ] `turtlecoin` v0.24.0
- [ ] `netbsd-tools` v9.0
- [x] `cwallpaper` v0.3.2
- [ ] `time` v1.9
- [ ] `rt-tests` v1.8
- [ ] `free42` v2.5.18
- [x] `minidlna` v1.2.1
- [x] `forked-daapd` v27.1
- [x] `libnspsl` v0.1.6
- [x] `libnslog` v0.1.3
- [x] `libharu` v2.3.0
- [x] `duktape` v2.5.0

## Packages to Drop

- [x] Pyside (Unmaintained, obsoleted by Pyside 2)
- [x] shiboken (Unmaintained, obsoleted by shiboken 2)
- [x] yubikey-piv-manager (Unmaintained, obsoleted by `yubikey-manager-qt`)
- [x] OpenSTF
  - [x] `openstf`
  - *Requires Node.js < 10.*
- [x] Shadowsocks
  - [x] `shadowsocks`
  - *Deprecated and unmaintained, users to be redirected to `shadowsocks-libev`.
- [x] `akonadi4`
- [x] `amor`
- [x] `anbox`
- [x] `arm`
- [x] `blogilo`
- [x] `byzanz`
- [x] `caddy`
- [x] `caffe-opencl`
- [x] `casuarius`
- [x] `cutorch`
- [x] `dgsh`
- [x] `enterprisecontrolconfigurationlogging`
- [x] `it87`
- [x] `jovie`
- [x] `js-17`
- [x] `js-31`
- [x] `js-45`
- [x] `juffed`
- [x] `kaccessible`
- [x] `kactivities4-runtime`
- [x] `kde-base-artwork`
- [x] `kde-runtime`
- [x] `kdepimlibs4`
- [x] `kdevplatform`
- [x] `kdiff3`
- [x] `kfilereplace`
- [x] `klinkstatus`
- [x] `kommander`
- [x] `kremotecontrol`
- [x] `kscd`
- [x] `libbson`
- [x] `libgltf`
- [x] `libkdeedu`
- [x] `libkdegames4`
- [x] `libkmahjongg4`
- [x] `libkolab`
- [x] `llvm-3.9`
- [x] `mysql-workbench`
- [x] `pairs`
- [x] `pcap-dnsproxy`
- [x] `pim-storage-service-manager`
- [x] `prison`
- [x] `pykde`
- [x] `python-prompt-toolkit`
- [x] `qtruby`
- [x] `qtspeech`
- [x] `qtstyleplugin`
- [x] `qtvirtualkeyboard`
- [x] `qupzilla`
- [x] `qwx`
- [x] `qyoto`
- [x] `rekonq`
- [x] `requests-toolbelt`
- [x] `riscv-qemu-user-static`
- [x] `rustfmt`
- [x] `shadowsocks`
- [x] `slimerjs`
- [x] `smokegen`
- [x] `smokekde`
- [x] `smokeqt`
- [x] `torch7-*`
- [x] `treefrog-framework`
- [x] `xmp-toolkit`

## OS-wide Projects

- [x] Meta: Re-evaluate All PKGRECOM/PKGSUG (#1918)
- [ ] Meta: Apply HTTPS to all packages where available (including `ftp://` sources).

## Help Wanted

N/A

## Documentation Work

- [Security updates for 2019](https://aosc.wiki/aosa/archive/2019)
- [Security updates for 2020](https://aosc.wiki/aosa/archive/2020)

## See Also

- [Open Security Issues](https://github.com/AOSC-Dev/aosc-os-abbs/issues?q=is%3Aissue+is%3Aopen+label%3Asecurity)
- [Repology](https://repology.org/repository/aosc)

## Deferred Items

- March 24, 2020: LLVM v10.0.0 [\[llvm-announce\] LLVM 10.0.0 Release](http://lists.llvm.org/pipermail/llvm-announce/2020-March/000087.html)

**Comment by MingcongBai** [comment]
Spring 2020 is now live.

**Comment by MingcongBai** [comment]
Frozen. Future work to be recorded in Summer 2020 iteration plan, when available. @KexyBiscuit
